src/Pages/Basket/Basket.js

```diff
@@ -0,0 +1,321 @@
+        });
+      }
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  requestDeleteItem = async orderItemId => {
+    if (orderItemId.length <= 0) {
+      console.error('아이템을 선택하세요');
+      return;
+    }
+
+    try {
+      const res = await fetch(`${API}/orders/cart`, {
+        method: 'PUT',
+        headers: {
+          Authorization: `${localStorage.getItem('access_token')}`,
+        },
+        body: JSON.stringify({
+          order_item_id: orderItemId,
+        }),
+      });
+      const result = await res.json();
+
+      if (result.message === 'DELETE_SUCCESS') {
+        this.deleteItems(orderItemId);
+      } else {
+        console.error('메시지가 올바르지 않습니다');
+      }
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  handleChangeCheckBox = () => {
+    this.setState(
+      {
+        isCheckedAllItems: !this.state.isCheckedAllItems,
+      },
+      this.checkAllItems
+    );
+  };
+
+  handleClickDeleteButton = () => {
+    this.requestDeleteItem(this.state.checkedItemsId);
+  };
+
+  checkAllItems = () => {
+    const { items, isCheckedAllItems } = this.state;
+    const checkedItemsId = items.map(item => item.order_item_id);
+
+    if (isCheckedAllItems) {
+      this.setState({
+        checkedItemsId,
+      });
+    } else {
+      this.setState({
+        checkedItemsId: [],
+      });
+    }
+  };
+
+  checkItems = (isChecked, orderItemId) => {
+    const { checkedItemsId } = this.state;
+    if (isChecked) {
+      this.setState({
+        checkedItemsId: [...checkedItemsId, orderItemId],
+      });
+    } else {
+      this.setState({
+        checkedItemsId: checkedItemsId.filter(
+          checkedItemId => checkedItemId !== orderItemId
+        ),
+      });
+    }
+  };
+
+  deleteItems = orderItemId => {
+    const filteredItems = this.state.items;
+    const checkedItemsId = this.state.checkedItemsId;
+
+    if (orderItemId.length === 1) {
+      const [itemId] = orderItemId;
+      this.setState({
+        items: this.state.items.filter(item => item.order_item_id !== itemId),
+        checkedItemsId: checkedItemsId.filter(
+          checkedItemId => checkedItemId !== itemId
+        ),
+      });
+      return;
+    }
+
+    for (let i = 0; i < filteredItems.length; i++) {
+      for (let j = 0; j < checkedItemsId.length; j++) {
+        if (
+          filteredItems[i] &&
+          filteredItems[i].order_item_id === parseInt(checkedItemsId[j])
+        ) {
+          filteredItems.splice(i, 1);
+          i--;
+        }
+      }
+    }
+
+    this.setState({
+      items: filteredItems,
+      checkedItemsId: [],
+    });
+  };
+
+  openQuantityForm = orderItemId => {
+    this.setState({
+      isClickedCountButton: !this.state.isClickedCountButton,
+      quantityFormId: orderItemId,
+    });
+  };
+
+  requestModifyQuantity = async (orderItemId, amount) => {
+    try {
+      const res = await fetch('http://10.58.1.215:8000/orders/cart', {
```

Comment:
```suggestion
      const res = await fetch(`${API}/orders/cart`, {
```
src/config에 url 주소 분리해두는게 좋을 것 같습니다

Reply:
ip주소 합쳐지는 대로 수정하겠습니다.